metadata/metadata.go

```diff
@@ -93,12 +93,16 @@ func (md MD) Copy() MD {
 }
 
 // Get obtains the values for a given key.
+//
+// All keys are converted to lowercase.
 func (md MD) Get(k string) []string {
 	k = strings.ToLower(k)
 	return md[k]
 }
 
 // Set sets the value of a given key with a slice of values.
+//
+// All keys are converted to lowercase.
```

Comment:
Similar: ~`k is converted to lowercase before storing in md.`

Reply:
Done